Mathlib/MeasureTheory/Integral/FiniteMeasureCharFun.lean

```diff
@@ -0,0 +1,223 @@
+/-
+Copyright (c) 2024 Jakob Stiefel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob Stiefel
+-/
+import Mathlib.Analysis.Complex.Circle
+import Mathlib.MeasureTheory.Measure.FiniteMeasureExt
+/-!
+
+# Characteristic Function of a Finite Measure
+
+This file defines the characteristic function of a `FiniteMeasure P` on a topological vector space
+`V`.
+
+## Probability Character
+
+We first define
+
+`probChar _ _ w : V →ᵇ ℂ := fun (v : V) ↦ e (L v w)`,
+
+where `e` is a continuous additive character and `L : V →ₗ[ℝ] W →ₗ[ℝ] ℝ` is a bilinear map. We
+then define `expPoly`, the star subalgebra of `ℂ`-linear combinations of elements of
+`{probChar _ _ w : w ∈ W}`. We show that `expPoly` separates points in `V`, see
+`expPoly_separatesPoints`.
+
+## Characterictic Function
+
+The characteristic function of a finite measure `P` on `V` is the mapping
+
+`fun w => ∫ v, e (-L v w) ∂P = ∫ v, probChar w ∂P`
+
+We show:
+- `MeasureTheory.ext_of_charFun_eq`: If the characteristic functions of two finite measures `P`
+  and `P'` are equal, then `P = P'`. In other words, characteristic functions separate finite
+  measures.
```

Comment:
Please see https://leanprover-community.github.io/contribute/doc.html for the formatting rules of file docstrings. In particular, we have standard sections like "## Main definitions" and "## Main statements".
If you type "module" in the file and look at auto-complete suggestions, you will get an option to generate a template module docstring with the correct sections (delete the sections that don't make sense).